### PR TITLE
UofT-DSI | Deep Learning - Assignment 2

### PR DESCRIPTION
**Pull request by klin093**
## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

File: 01_materials/labs/lab_3.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_3.ipynb:1]: **Exercises**: You defined two `Dropout` layers in the `RegressionModel`, but they were not used in the training. So the model you trained here is the same as the baseline model. Also, you reported `MAE` and `MSE` for the test samples. One of the suggested options was to **train** the model with a different loss function, e.g., `MSE`.

**Review by tianyi21 — APPROVED** [review]
Hi Kirby,

Thank you for submitting your DL week 2 assessment. Please find my comments attached to each file. Your A2, Lab3-6 are now complete.

All your DL assignments have been completed. Thank you for your participation, and I wish you all the best for your future!

Thanks,
Tianyi [LS]
